src/optimization/steepest_descent_method.cpp

```diff
@@ -20,282 +20,282 @@ namespace optimization
 {
 
 SteepestDescentMethod::SteepestDescentMethod(
-	double alpha,
-	double increment_of_the_argument,
-	size_t max_iteration_number,
-	double precision)
-	: alpha_(alpha)
-	, increment_of_the_argument_(increment_of_the_argument)
-	, max_iteration_number_(max_iteration_number)
-	, precision_(precision)
+    double alpha,
+    double increment_of_the_argument,
+    size_t max_iteration_number,
+    double precision)
+    : alpha_(alpha)
+    , increment_of_the_argument_(increment_of_the_argument)
+    , max_iteration_number_(max_iteration_number)
+    , precision_(precision)
 {
 }
 
 SteepestDescentMethod* SteepestDescentMethod::Clone() const
 {
-	return new SteepestDescentMethod(
-		alpha_,
-		increment_of_the_argument_,
-		max_iteration_number_,
-		precision_);
+    return new SteepestDescentMethod(
+        alpha_,
+        increment_of_the_argument_,
+        max_iteration_number_,
+        precision_);
 }
 
 std::string SteepestDescentMethod::get_name() const
 {
-	return "Steepest Descent Method";
+    return "Steepest Descent Method";
 }
 
 
 #ifdef BUILD_WITH_MPI
 void SteepestDescentMethod::Send(boost::mpi::communicator* communicator, int target) const
  {
-	communicator->send(target, k52::parallel::mpi::constants::kCommonTag, increment_of_the_argument_);
-	communicator->send(target, k52::parallel::mpi::constants::kCommonTag, max_iteration_number_);
-	communicator->send(target, k52::parallel::mpi::constants::kCommonTag, precision_);
+    communicator->send(target, k52::parallel::mpi::constants::kCommonTag, increment_of_the_argument_);
+    communicator->send(target, k52::parallel::mpi::constants::kCommonTag, max_iteration_number_);
+    communicator->send(target, k52::parallel::mpi::constants::kCommonTag, precision_);
 }
 
 void SteepestDescentMethod::Receive(boost::mpi::communicator* communicator, int source)
 {
-	communicator->recv(source, k52::parallel::mpi::constants::kCommonTag, increment_of_the_argument_);
-	communicator->recv(source, k52::parallel::mpi::constants::kCommonTag, max_iteration_number_);
-	communicator->recv(source, k52::parallel::mpi::constants::kCommonTag, precision_);
+    communicator->recv(source, k52::parallel::mpi::constants::kCommonTag, increment_of_the_argument_);
```

Comment:
PavelKovalets: 
> alfa is not transmitted, but it will be ok if it will be const (if it is a proven optimal value - provide a link to proof source)

Const in header?
[Link](https://ru.wikipedia.org/wiki/%D0%9C%D0%B5%D1%82%D0%BE%D0%B4_%D0%B7%D0%BE%D0%BB%D0%BE%D1%82%D0%BE%D0%B3%D0%BE_%D1%81%D0%B5%D1%87%D0%B5%D0%BD%D0%B8%D1%8F)